README.md

```diff
@@ -15,37 +15,92 @@ Argument Parser for C++20
 ## DEV NOTES
 
 1. Requirements:
-   * g++ >= g++-11
+   * g++ >= g++-20
```

Comment:
This should be 11, as this is the version of the GNU G++ compiler and not the language standard

Reply:
Done.